notebooks/link_analysis/Pagerank_compat.ipynb

```diff
@@ -0,0 +1,871 @@
+       " ('1', '8'),\n",
+       " ('14', '2'),\n",
+       " ('14', '4'),\n",
+       " ('24', '26'),\n",
+       " ('28', '25'),\n",
+       " ('34', '10'),\n",
+       " ('34', '14'),\n",
+       " ('34', '15'),\n",
+       " ('34', '16'),\n",
+       " ('34', '19'),\n",
+       " ('34', '20'),\n",
+       " ('34', '21'),\n",
+       " ('34', '23'),\n",
+       " ('34', '24'),\n",
+       " ('34', '27'),\n",
+       " ('34', '28'),\n",
+       " ('34', '29'),\n",
+       " ('34', '30'),\n",
+       " ('34', '31'),\n",
+       " ('34', '32'),\n",
+       " ('34', '33'),\n",
+       " ('34', '9'),\n",
+       " ('9', '1'),\n",
+       " ('9', '3')]"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sorted(list(nx.bfs_tree(Gnx,source='34',depth_limit=3).edges()))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Starting with a NetworkX implementation and switching to cugraph algorithms is that easy, just an import change.\n",
+    "The start to finish cugraph implementation is included below for reference.\n",
+    "\n",
+    "----"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# cuGraph"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Read in the data - GPU\n",
+    "cuGraph graphs can be created from cuDF, dask_cuDF and Pandas dataframes\n",
+    "\n",
+    "The data file contains an edge list, which represents the connection of a vertex to another.  The `source` to `destination` pairs is in what is known as Coordinate Format (COO).  In this test case, the data is just two columns.  However a third, `weight`, column is also possible"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import required libraries\n",
+    "import cudf\n",
+    "import cugraph"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read the data  \n",
+    "gdf = cudf.read_csv(datafile, names=[\"src\", \"dst\"], delimiter='\\t', dtype=[\"int32\", \"int32\"] )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a Graph "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a Graph using the source (src) and destination (dst) vertex pairs from the Dataframe \n",
+    "G = cugraph.from_edgelist(gdf, source='src', destination='dst')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Call the PageRank algorithm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Reached pagerank !!!!\n"
```

Comment:
I think the output text (especially internal debugging output like this) should not be included.  I think we keep output text for benchmark notebooks that take a long time to run so users can see the results, but this one might benefit from being reset before committing.  You should be able to reset all cells in jupyter before saving and committing.